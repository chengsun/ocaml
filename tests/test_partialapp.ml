let fma x y z = x * y + z

let three_ma = fma 3
let add10 = fma 5 2
let add6 = three_ma 2

let () = (
  Printf.printf "%d\n" (three_ma 100 33);
  Printf.printf "%d\n" (add10 1);
  Printf.printf "%d\n" (add6 60);
)
