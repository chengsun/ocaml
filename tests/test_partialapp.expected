333
11
66
